cmd/mt-store-cat/main.go

```diff
@@ -266,22 +293,28 @@ func main() {
 			fmt.Printf("metric id %v not found", amkey.MKey)
 			return
 		}
-		for _, m := range metrics {
-			fmt.Println(m)
+		if verbose {
```

Comment:
i'd just create some wrapper function `logIfVerbose()`. I guess we don't need to worry about an extra function call here